docs/ovs_communication.adoc

```diff
@@ -43,12 +43,32 @@ TBD - XiaoDong
 
 How do we call into OVS, what is the exact command we use to program a new port? What is the port configuration info needed?
 
+A: we would use ovs-vsctl add-port cmd with lots of options to add or to configure (set port)
```

Comment:
have we closed on to use ovs-vsctl to talk to OVS? Are there other options? Does neutron agent use ovs-vsctl command directly?

Reply:
I think we could use ovs-vsctl or using library like here the idl way or ofctl.api.send_msg way (OSKen) to operate

Reply:
1. ovs-vsctl
2. idl binding like this 
3. ofctl.api.send_msg (packet)